mathics/builtin/atomic/strings.py

```diff
@@ -486,6 +486,7 @@ class CharacterEncodings(Predefined):
     <url>
     :WMA link:
```

Comment:
I think a backslash is missing here

Reply:
No backslash is needed inside a link between `<url>` and the tag name `:WMA link:`